docs/modules/ROOT/pages/runbooks/ingress.adoc

```diff
@@ -0,0 +1,205 @@
+|  | lb 2    |      |      |              |                  |
+|  |         |      |      +--------------|------------------+
+|  |         |      |                     |
++->|         |------/      +------------+ | +----------------+
+\-------------------\      | master     | | | worker         |
+   |         |      |      | /--------\ | | |   /-------\    |
+   +---------+      |      | |        | | | |   |       |    |
+                    \---=----|Operator| | \---->|Canary |    |
+                           | |        | |   |   |       |    |
+                           | \--------/ |   |   \-------/    |
+                           |            |   |                |
+                           +------------+   +----------------+
+
+....
+
+There is a simple canary target running on every worker and infrastructure node.
+Every minute the ingress operator sends a probe to the external address of the canary target.
+This means it will send a request to the public floating IP of the load balancers, which will forward the request to one of the ingress controller running on the infrastructure nodes, which will then forward the request to one of the canary targets.
+
+Any of these steps might fail and lead to a failed canary probe.
+
+
+There are two types of alerts that fire if we expect to miss the configured objective.
+
+* A ticket alert means that the error rate is slightly higher than the objective.
+If we don't intervene at some point after receiving this alert, we can expect to miss the objective.
+However no immediate, urgent action is necessary.
+A ticket alert should have a label `severity: warning`.
+* A page alert means that the error rate is significantly higher than the objective.
+Immediate action is necessary to not miss the objective.
+A page alert should have a label `severity: critical` and should page on-call.
+
+
+=== icon:bug[] Steps for debugging
+
+Probe failures can have multiple reasons and the impact of this alert might vary.
+There are generally four possible causes for this failure:
+
+* *Load Balancer Failure*: There might be an issue with the load balancer VMs or its floating IP.
+This can include issues with inbound or outbound connections.
+* *Overlay Network Failure*: There might be an issue with the overlay network.
+In this case you probably see a lot of other issues.
+* *Ingress Controller Failure*: The ingress controller might have failed.
+* *Canary Target Failure*: The issue might also just be that the canary targets aren't available.
+In this case this isn't really an issue with the cluster ingress.
+
+Given that the cluster was able to send this alert, a complete outage of the load balancers or the overlay network is unlikely.
+Here are a few first steps you can do to narrow down the issue.
+
+First try to reproduce that you can't reach the canary endpoint yourself.
+Open the canary URL `canary-openshift-ingress-canary.$APP_DOMAIN` from your browser.
```

Comment:
Can we add the canary URL to the alert description? Or is it already part of it anyway?

Reply:
I don't think it is, but that is a good idea.